tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import re
+
+import pytest
+import random
+
+from tests.helpers import get_worker_nodes
+
+from ocs_ci.framework.pytest_customization.marks import tier4b
+from ocs_ci.ocs.resources import pod
+
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs import node
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4b
```

Comment:
please change to tier4a as this tier is being executed more frequently and we'd like these tests to be there

Reply:
Done